app/views/Notification.js

```diff
@@ -159,21 +192,44 @@ class NotificationScreen extends Component {
               </View>
 
               <ScrollView contentContainerStyle={styles.contentContainer}>
-                {this.state.data.map(data => (
-                  <View key={data.x} style={styles.notificationView}>
-                    <Text
-                      style={[
-                        styles.notificationsText,
-                        data.y > 80
-                          ? styles.notificationsTextRed
-                          : data.y > 50
-                          ? styles.notificationsTextOrange
-                          : null,
-                      ]}>
-                      {'Day ' + data.x + ': ' + data.y + ' intersections'}
-                    </Text>
-                  </View>
-                ))}
+                {this.state.data.map(data =>
+                  data.y === 0 ? (
+                    data.x == 13 && !hasExposure ? (
+                      <Text style={styles.noExposure}>
+                        No known exposures to COVID during the last two weeks.
+                      </Text>
+                    ) : (
+                      <Text style={{ height: 0 }}></Text>
+                    )
+                  ) : (
+                    <View key={data.x} style={styles.notificationView}>
+                      <Text
+                        style={[
+                          styles.notificationsText,
+                          data.y > 80
+                            ? styles.notificationsTextHigh
+                            : data.y > 50
+                            ? styles.notificationsTextMedium
+                            : null,
+                        ]}>
+                        {data.x == 0
+                          ? languages.t(
+                              'label.notifications_exposure_format_today',
+                              { exposureTime: data.y * 5 },
```

Comment:
What is 5? 5 minutes? 5 days? use a const

Reply:
const'antly complaining ... ;)